src/main/java/org/example/annotations/Table.java

```diff
@@ -0,0 +1,37 @@
+package src.main.java.org.example.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Table {
+    String name() default "";
+
+    String catalog() default "";
+
+    String schema() default "";
+
+    UniqueConstraint[] uniqueConstraints() default {};
+
+    Index[] indexes() default {};
+
+}
+@Target({})
```

Comment:
Don't you think it would be better to separate these three annotations in three different files.

Reply:
Yup, on it